.gitignore

```diff
@@ -9,4 +9,5 @@ _vendor*/
 /cli
 data/
 .vscode/
-.idea/
\ No newline at end of file
+.idea/
+**/*.iml
```

Comment:
worth to add to global git ignore, no?

Reply:
sure you can do that too, this does no harm though for anyone that hasn't done so.